### PR TITLE
Add volume mapping for OneRepoToRuleThemAll in docker compose.

### PR DESCRIPTION
**Pull request by akath20**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -10,7 +10,7 @@ const server = new WebpackDevServer(webpack(config), {
   disableHostCheck: true,
 });
 
-server.listen(process.env.PORT || 5000, 'localhost', function (err) {
+server.listen(process.env.PORT || 5000, '0.0.0.0', function (err) {
```
Comment (akath20) [inline server.js]: @billdybas any concerns with this?

Comment (wadeanthony0100) [reply server.js]: Is there a reason to prefer `0.0.0.0` over `localhost`?

Comment (akath20) [reply server.js]: Docker requires this to port outside the container, didn’t really affect users using node as long as we change the output text.

Comment (billdybas) [reply server.js]: `0.0.0.0` is the same thing as `localhost` except it opens the port to your network. So for example, if you knew the IP address of your computer, you could access your local server from your phone by going to that IP address w/ the port specified.

Seems fine to me. I think going to `localhost` w/ it also works, no?

**Pull request merged**